ios/browser/api/ads/brave_ads.mm

```diff
@@ -268,6 +271,7 @@ - (void)initialize:(void (^)(bool))completion {
   adEventHistory = new ads::AdEventHistory();
 
   adsClient = new AdsClientIOS(self);
+  adsClientObserverNotifier = new AdsClientObserverNotifier();
```

Comment:
What does this notifier do? I don't see any connection to the original `ads` class which had these methods before. Is there a binding/init of some sort that we're missing?

Reply:
Ditto